.travis.yml

```diff
@@ -15,12 +15,12 @@ before_install:
 
 script:
   - GO111MODULE=on go test ./...
-  - docker build --tag kube-bench .
-  - docker run -v `pwd`:/host kube-bench install 
+  - docker build  --tag kube-bench .
+  - docker run -v `pwd`:/host -u 0:0 kube-bench install
```

Comment:
The -u 0:0 explicitly sets the user to root, overriding what is set in the image